sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuColumnarToRowExec.scala

```diff
@@ -343,6 +343,16 @@ object GpuColumnarToRowExecParent {
     Option(Tuple2(arg.child, arg.exportColumnarRdd))
   }
 
+  /**
+   * Helper to check if GPU accelerated row-column transpose is supported
+   */
+  private def isAcceleratedTransposeSupported: Boolean = {
```

Comment:
We can cache the value rather than continually detecting the version:
```suggestion
  private lazy val isAcceleratedTransposeSupported: Boolean = {
```

Reply:
That's where I first started. I might have overthought it:

Could the Exec instance be shared in case the same machine had two GPUs of different architectures?

Reply:
The RAPIDS Accelerator only supports one GPU per executor.  The executor picks a GPU device on startup and sticks with it.

Reply:
Fixed.